changelog.d/10477.bugfix

```diff
@@ -0,0 +1 @@
+Fix circular import with the default SAML mapping handler.
```

Comment:
Is this an issue when using SAML or only when sub-classing the default mapping handler or something else?

Reply:
I noticed this issue with an EMS host that was configured without a custom mapping handler, so it tried to import the default one when we run the config test before deploying.

We have some custom mapping providers which don't subclass the default one.